src/expressions.jl

```diff
@@ -95,7 +94,7 @@ function size(x::AbstractExpr)
 end
 
 function length(x::AbstractExpr)
-    return prod(x.size)
+    return x.size[1] * x.size[2]
```

Comment:
This change generates the exact same code. I would leave it as is.

Reply:
Okay